pwiz_tools/Skyline/SettingsUI/ViewLibraryPepInfoList.cs

```diff
@@ -29,6 +29,26 @@
 
 namespace pwiz.Skyline.SettingsUI
 {
+    // Workaround for missing String.Contains(String, StringComparison)
+    // cf. https://learn.microsoft.com/en-us/dotnet/api/
+    // system.string.contains?view=netframework-4.7.2#system-string-contains(system-string-system-stringcomparison)
+    // Remove this if ever moving to .NET Core!
+    public static class StringExtensions
+    {
+        public static bool Contains(this String str, String substring,
+                                    StringComparison comp)
+        {
+            if (substring == null)
+                throw new ArgumentNullException("substring",
+                                             "substring cannot be null.");
+            else if (!Enum.IsDefined(typeof(StringComparison), comp))
+                throw new ArgumentException("comp is not a member of StringComparison",
+                                         "comp");
+
+            return str.IndexOf(substring, comp) >= 0;
+        }
+    }
+
```

Comment:
This is nice, but probably overkill - just using IndexOf directly is cleaner and easier to understand.